src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -122,6 +123,16 @@ if (__DEV__) {
     warning(false, 'Extra attributes from the server: %s', names);
   };
 
+  var warnForInvalidEventListener = function(registrationName, listener) {
+    warning(
+      false,
+      'Expected %s listener to be a function, instead got type %s%s',
```

Comment:
Nit: I think we usually end warnings with periods (before the stack) so that they look like sentences.